src/js/helpers/graph/declarationHelper.js

```diff
@@ -86,7 +86,10 @@ const createVirtualEndNode = (composite, transitions) => ({
 });
 
 const createTransitionsToSubtasks = (subtasks) => subtasks
```

Comment:
Out of curiosity: Why do you make subtasks an object instead of array?

Reply:
Because this whole function creates a graph that do not distinct between subtasks and normal nodes so if they would be an array I would have to have a check for that in a few different places in the code